### PR TITLE
Update PeopleList to use ListBackgroundView

### PR DESCRIPTION
**Pull request by vanwagonet**
extra: update panda images to latest designs

refs: none
affects: Student, Teacher
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by vanwagonet** [comment]
This should make PeopleList our current gold standard for lists. It handles search, loading state, empty state, error state, paging, & pull to refresh.

Test Plan: People list in Student & Teacher should work great.

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/jummglu) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fjummglu)
teacher: [Link](https://rebrand.ly/ax7r0ed) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fax7r0ed)
parent: [Link](https://rebrand.ly/p3k3hdl) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fp3k3hdl)

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1024303&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student, Teacher

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.62% | 93.22% | 0.4%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/People/PeopleListViewController.swift [hunk 1]
```diff
@@ -64,14 +68,16 @@ public class PeopleListViewController: UIViewController, ColoredNavViewProtocol,
         view.backgroundColor = .named(.backgroundLightest)
         setupTitleViewInNavbar(title: NSLocalizedString("People", bundle: .core, comment: ""))
 
-        activityIndicatorView.color = Brand.shared.primary
-        emptyResultsLabel.text = NSLocalizedString("No results", bundle: .core, comment: "")
+        emptyMessageLabel.text = NSLocalizedString("We couldn’t find somebody like that.", bundle: .core, comment: "")
+        emptyTitleLabel.text = NSLocalizedString("No Results", bundle: .core, comment: "")
+        errorView.messageLabel.text = NSLocalizedString("There was an error loading people. Pull to refresh to try again.", bundle: .core, comment: "")
+
```
Comment (armstrongnate) [inline Core/Core/People/PeopleListViewController.swift]: I believe the error view retry button needs to be hooked up.
```suggestion
errorView.retryButton.addTarget(self, action: #selector(refresh), for: .primaryActionTriggered)

```

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1024334&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1024335&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**